package.json

```diff
@@ -42,12 +42,12 @@
     "test": "./gradlew testServerDebugUnitTest"
   },
   "devDependencies": {
-    "@appium/eslint-config-appium-ts": "^1.0.0",
-    "@appium/support": "^6.0.0",
+    "@appium/eslint-config-appium-ts": "^2.0.0-rc.1",
+    "@appium/support": "^7.0.0-rc.1",
     "@semantic-release/changelog": "^6.0.1",
     "@semantic-release/git": "^10.0.1",
     "android-apidemos": "^4.0.0",
```

Comment:
should we bump apidemos first?

Reply:
We could but the npm package's JS is to build the test app, so technically it doesn't matter. Let me bump the apidemos also.